tests/e2e/testsuites/modify_volume_tester.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+   http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testsuites
+
+import (
+	"context"
+	"fmt"
+	"github.com/kubernetes-sigs/aws-ebs-csi-driver/tests/e2e/driver"
+	. "github.com/onsi/ginkgo/v2"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
+
+// ModifyVolumeTest will provision pod with attached volume, and test that modifying its pvc will modify the associated pv.
+type ModifyVolumeTest struct {
+	CreateVolumeParameters                map[string]string
+	ModifyVolumeAnnotations               map[string]string
+	ShouldResizeVolume                    bool
+	ShouldTestInvalidModificationRecovery bool
+}
+
+const (
+	// TODO Q: This `modifyVolumeTest` prefix is a code smell. Does it signify that `testsuites` package should be broken up into separate testsuite packages importing a shared testsuite-utils package?
+	modifyVolumeTestDefaultVolumeSize = "10Gi" // Different from driver.MinimumSizeForVolumeType to simplify iops, throughput, volumeType modification
```

Comment:
Suggest moving this const down to the var() segment and renaming to `volumeSize`.